utils/deployer.ts

```diff
@@ -86,54 +95,54 @@ export default class ContractDeployer {
         libraries?: Libraries | undefined
     }): Promise<DeployResult> {
         const {contract, name, proxy, args, libraries} = config
-        const targetName = `${name}Target`
 
+        const shouldRegister = this.controller && !this.skipRegister
```

Comment:
Feels like this function should be renamed to avoid confusion i.e. calling deployAndRegister() but registration does not occur due to the ContractDeployer being configured with skipRegister = true.

Just `deploy()`?

Reply:
Implemented on https://github.com/livepeer/protocol/pull/631/commits/fccc671a90258b168b65aba0a0356741876a430f LMK WDYT!